R/fillsink.R

```diff
@@ -0,0 +1,36 @@
+#' wrap_fillsink
+#'
+#' This will make the fillsinks funktion available to R from the libtotopotoolbox subdirectory
+#' 
+#' @param dem Takes a Spatrast as input 
+#' @param bc Array of the same size as the input matrix
+#'
+#' @return A Spatrast
+#' @export
+
+fillsink <- function(dem,bc=NULL) {
```

Comment:
Let's change the name to `fillsinks` to be consistent with libtopotoolbox and our other packages.